### ISSUE TITLE
Wallpaper not filling the whole screen / Workaround for border issue after updating to Windows 11 24H2

### ISSUE THREAD

**Issue by TheQuickFox** [rocksdanister/lively#2415]
When you get the top and right of your screen not being updated with the wallpaper animation in Lively after updating to Windows 11 24H2, try this workaround:

              Try creating a `mpv.conf` file
```
border=no
```
and save it in the following folder:
https://github.com/rocksdanister/lively/wiki/Video-Guide#mpv

_Originally posted by @rocksdanister in https://github.com/rocksdanister/lively/issues/2108#issuecomment-2146805795_

**Comment by rocksdanister** [comment]
This issue started happening from mpv build 2024-03-31 https://github.com/rocksdanister/lively/issues/2074#issuecomment-2146541025

> There is an alignment issue with Mpv player (video wallpaper) in 24h2 beta builds:
![image](https://github.com/rocksdanister/lively/assets/17554161/e5cc9a05-6be6-4402-9109-4c719d453e2f)
The issue started happening from Mpv build `2024-03-31` onwards:
https://sourceforge.net/projects/mpv-player-windows/files/64bit/
Switching to the previous build from 2024-03-17 `mpv-x86_64-20240317-git-3afcaeb.7z` fixed the issue:
https://sourceforge.net/projects/mpv-player-windows/files/64bit/mpv-x86_64-20240317-git-3afcaeb.7z/download

Or just create the `mpv.conf` file.
Or just install the Microsoft Store version of Lively which is already patched.

**Comment by rocksdanister** [comment]
Can you check in 24h2 if the issue exists with the latest x64 mpv release with and without the border flag.

**Comment by TheQuickFox** [comment]
> Can you check in 24h2 if the issue exists with the latest x64 mpv release with and without the border flag.

This is with the latest release from your releases page.
https://github.com/rocksdanister/lively/releases

**Comment by rocksdanister** [comment]
No, Mpv from sourceforge: https://github.com/rocksdanister/lively/issues/2074#issuecomment-2146541025

**Comment by TheQuickFox** [comment]
~~https://sourceforge.net/projects/mpv-player-windows/files/64bit/mpv-x86_64-20240929-git-c3d9243.7z/download~~

~~Fixes it~~

**Comment by rocksdanister** [comment]
Just to make sure this is with the mpv.conf file deleted right?

**Comment by TheQuickFox** [comment]
> Just to make sure this is with the mpv.conf file deleted right?

![mpv](https://github.com/user-attachments/assets/3a1a9f81-ed1b-41ba-bb87-22cddad22e32)

**Comment by TheQuickFox** [comment]
> ~https://sourceforge.net/projects/mpv-player-windows/files/64bit/mpv-x86_64-20240929-git-c3d9243.7z/download~
> 
> ~Fixes it~

**Update**: It does not. After the second reboot the issue is back. My workaround still works. For now I'm going to use that. I still got 5 PC's to upgrade from Windows 11 23H2 to 24H2.

**Comment by rocksdanister** [comment]
Looks like I wont be getting the update anytime soon
https://www.pcmag.com/news/microsoft-windows-11-24h2-intel-issues-may-causes-games-to-crash

**Comment by TheSlowFox** [comment]
The mpv.conf does not reliably solve the issue for me, while the MPV build from 2024-03-17 does.

**Comment by rocksdanister** [comment]
Are you sure the mpv.conf file was read by mpv? You can try adding `osd-level=3` for testing https://mpv.io/manual/stable/#options-osd-level

**Comment by TheSlowFox** [comment]
Good question: my editor had silently added a .txt suffix to the mpv.conv, so MPV could not find it...
With the correct file name, it now works with old and new MPV builds. Thanks!

**Comment by lv0203** [comment]
ok，there is an another method from chatgpt.For the moment,it's effective for me.

Check for Compatibility Mode
Right-click on the Lively Wallpaper shortcut or executable.
Select Properties > Compatibility.
Enable Run this program in compatibility mode for and select a previous version of Windows (e.g., Windows 8).Recommend selecting the first three options under 'Settings' (i.e., Disable Full Optimization, Run this program as administrator, Register this program to restart)."  
This can sometimes resolve display issues after a major update.

**Comment by rocksdanister** [comment]
Do not change compatibility settings. The three solutions listed are recommended.

**Comment by Clannad06** [comment]
Thanks and sorry for the duplicate #2609

**Comment by MagmaSlime123** [comment]
Maybe this fix should be included in the actual program during install, so manual intervention by the user isn't required.
(It's 1 file. Not hard to add to an install process.)

**Comment by rocksdanister** [comment]
This is already resolved https://github.com/rocksdanister/lively/commit/7cc4116f5197bf02115d8eec5c5125942b804d52

> This issue started happening from mpv build 2024-03-31 [#2074 (comment)](https://github.com/rocksdanister/lively/issues/2074#issuecomment-2146541025)
> 
> > There is an alignment issue with Mpv player (video wallpaper) in 24h2 beta builds:
> > ![image](https://github.com/rocksdanister/lively/assets/17554161/e5cc9a05-6be6-4402-9109-4c719d453e2f)
> > The issue started happening from Mpv build `2024-03-31` onwards:
> > https://sourceforge.net/projects/mpv-player-windows/files/64bit/
> > Switching to the previous build from 2024-03-17 `mpv-x86_64-20240317-git-3afcaeb.7z` fixed the issue:
> > https://sourceforge.net/projects/mpv-player-windows/files/64bit/mpv-x86_64-20240317-git-3afcaeb.7z/download
> 
> Or just create the `mpv.conf` file. Or just install the Microsoft Store version of Lively which is already patched.

Installer version just needs to wait for the update.
